src/antlr/Parser.g

```diff
@@ -1814,6 +1814,7 @@ relation[WhereClause.Builder clauses]
     | name=cident '[' key=term ']' type=relationType t=term { $clauses.add(Relation.mapElement(name, key, type, t)); }
     | ids=tupleOfIdentifiers
         ( K_IN inValue=multiColumnInValues { $clauses.add(Relation.multiColumn(ids, Operator.IN, inValue)); }
+        | K_NOT K_IN inValue=multiColumnInValues { $clauses.add(Relation.multiColumn(ids, Operator.NOT_IN, inValue)); }
```

Comment:
Can this be combined with the previous line by using `inOperator`? As in:
```
( rt=inOperator inValue=multiColumnInValues { $clauses.add(Relation.multiColumn(ids, rt, inValue)); }
```

Reply:
Yes, that was the intention but I changed it only on 1807, good catch, thanks!